src/components/dialogs/ConfirmDissolution.vue

```diff
@@ -56,28 +57,64 @@ export default class ConfirmDissolution extends Vue {
 
   // Global getters
   @Getter getEntityName!: string
-  @Getter isCoop!: boolean
+  @Getter getEntityType!: string
 
   /** The entity title to display. */
   private get entityTitle (): string {
-    return this.isCoop ? 'Cooperative Association' : 'Company'
+    return this.getEntityDetails().entityTitle
   }
 
   /** The entity title to display. */
   private get subEntityTitle (): string {
-    return this.isCoop ? 'cooperative' : 'company'
+    return this.getEntityDetails().subTitle
   }
 
   /** The entity title to display. */
   private get entityAct (): string {
-    return this.isCoop ? 'Cooperative Association' : 'Business Corporations'
+    return this.getEntityDetails().act
+  }
+
+  /** The entity title to display. */
+  private get getModalTitle (): string {
+    return this.getEntityDetails().title
   }
 
   // Pass click event to parent.
   @Emit() private close () { }
 
   // Pass click event to parent.
   @Emit() private proceed () { }
+
+  // returning  title subtitle, and act by entityType
+  private getEntityDetails (): any {
+    let entityTitle, subTitle, act, title
+
+    switch (this.getEntityType) {
+      case CorpTypeCd.COOP:
+        entityTitle = 'Cooperative Association'
+        subTitle = 'cooperative'
+        act = 'Cooperative Association'
+        break
+      case CorpTypeCd.SOLE_PROP:
+        entityTitle = 'Sole Proprietorship'
+        subTitle = 'registered business'
+        act = 'Partnership'
+        title = 'Dissolution'
+        break
+      case CorpTypeCd.PARTNERSHIP:
+        entityTitle = 'Partnership'
+        subTitle = 'registered business'
+        act = 'Partnership'
+        title = 'Dissolution'
+        break
+      default:
+        entityTitle = 'CompanyCompany'
+        subTitle = 'company'
+        act = 'Business Corporations'
+        title = 'Voluntary Dissolution'
+    }
+    return { entityTitle, subTitle, act, title }
+  }
```

Comment:
This project has not yet implemented the "resources concept" as per Create UI or Edit UI. We knew this was coming but I didn't know you would be the first to really benefit from this.

Would you be able to copy the resources concept here please? In a nutshell:
- in a "resources" folder are resource files for each entity type, eg: https://github.com/bcgov/business-create-ui/tree/main/src/resources/Incorporations (you probably don't need the sub-folder)
- in this component you can "find" the correct resources, eg: https://github.com/bcgov/business-create-ui/blob/12d332edb99ff14a4fb6c729c2be0e8926d2de59/src/App.vue#L774 (up to you whether the resources need to be global/store or not; we can refactor this later as needed)
- in this component, just display the labels from the resource

Reply:
As discussed in offline moved to business-config

Reply:
It looks waaaaay better :)